bench/wtperf/wtperf.c

```diff
@@ -762,6 +762,90 @@ worker(void *arg)
 					value_buf[0] = 'a';
 				if (opts->random_value)
 					randomize_value(thread, value_buf);
+
+				/*
+				 * Distribute the modifications across
+				 * whole document
+				 */
+				if (opts->distribute_modify)
+				{
```

Comment:
If `modify` becomes a workload thread operation type a lot of these indented conditionals would go away I think.

Reply:
corrected.